CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - Add new `RSpec/Rails/MinitestAssertions` cop. ([@ydah])
 - Fix a false positive for `RSpec/PendingWithoutReason` when not inside example. ([@ydah])
 - Fix a false negative for `RSpec/PredicateMatcher` when using `include` and `respond_to`. ([@ydah])
+- Fix a false negative for `RSpec/Pending` when using skipped in metadata is multiline string. ([@ydah])
+- Fix a false positive for `RSpec/NoExpectationExample` when using skipped in metadata is multiline string. ([@ydah])
```

Comment:
[memo] CHANGELOGs are separated for easy reading. Please let us know if it would be better to integrate.